### ISSUE TITLE
Newsletter font

### ISSUE THREAD

**Issue by YogiJohn** [codewars/codewars.com#2089]
#### Describe the bug
Since switching to a new computer (Win 10 Pro) and moving to Outlook 2019 (Home and Office) the text of each Kata described in the newsletter appear similar to an underline (exception of some text that appears to be labeled with the code tag, such as the n in the screenshot).

Copying and pasting from the email source into Word I notice the 'underlines' are Calibri font in a 1-pt size.  The n in the screenshot is Courier New, 10-pt.

Based on saving the email source as a .xml file and opening it in Word, the offending line appears to be located within the  <!--[if gt mso 15]> block:

```
/* Outlook 2016 Height Fix */   table, tr, td { border-collapse: collapse; }   tr { font-size:0px; line-height:0px; border-collapse: collapse; }
```

Word shows the same as Outlook.  However, if I move the */ to the end of the line everything appears fine when I reopen it in Word.

#### To Reproduce

Open a weekly newsletter in Outlook 2019.

(May only be affecting my computer(?) and is possibly related to a default font setting.)

#### Expected Behavior

See the newsletter as I did under Outlook 2010.

#### Screenshots

From 8 Apr 2020 newsletter:
![image](https://user-images.githubusercontent.com/8044008/78805405-39243f00-7976-11ea-96b5-7e53a80d2e5f.png)

#### Contexts

Please complete the following information:

- Codewars username (if account related): [e.g., jhoffner]
- OS: Win 10
- Browser: n/a (Outlook 2019)
- Browser Version: [e.g. 22]

**Comment by kazk** [comment]
Thanks for opening an issue here. I got the email and briefly looked into it, but I had no idea what's going on.

**Comment by YogiJohn** [comment]
What tool is used to generate the newsletter?  Can the html it generates be modified?  This may be an issue to pass on to the company in charge of that tool.  The Outlook 2016 Height Fix appears to be related to a line height issue that appeared in Outlook 2016.  I'm not sure why the html generated for your newsletter specifies font-size:0px.

As for possible fixes, delete the font-size:0px in the relevant block or change [if gt mso 15] test to [if eq mso 16]. 

An important question, though, is what anyone else using Outlook 2019 sees.  Perhaps a message in the next newsletter to the effect "If you use Outlook 2019 please look at this bug report..."  If others see it correctly then I need to be looking at my settings (Word renders the email; I'll look into that).

**Comment by YogiJohn** [comment]
I'm not positive what's going on, but it seems as though the tool you use to write the message inserts html code for a known Outlook 2016 Height Fix (where Outlook defaults to TDs having a minimum 17px height) but sets the font-size to 0 px as well as the height.  This may be an issue to forward to the developers of the css or tool used to generate the newsletter.  First recommendation is to try eliminating the font-size:0px entry in the fix; second recommendation would be for the enclosing block to apply the fix only if ms0 equals 16 (currently looks for mso greater than 15).

I can see the message fine in webmail and in Outlook 2010.  In webmail I can forward it to myself (inline; I can't attach it) and I can view it fine in Outlook 2019 (a new style sheet seems to apply; there is no Height Fix code).

I'm happy to help with testing.  If you use another company's software to generate the newsletter feel free to give them my contact info.

**Comment by kazk** [comment]
Thanks for looking into it. I need to ask my colleague for the details since it's done outside of the app and I'm not involved with the process.
I'll let you know once I find out.

**Comment by kazk** [comment]
The newsletter is sent using [MailerLite](https://www.mailerlite.com/)'s standard template. I don't think we can make changes to the injected code, but I'm not sure.

**Comment by YogiJohn** [comment]
MailerLite seems to have multiple templates (https://www.mailerlite.com/newsletter-examples) and it appears you can customize one of their templates (https://help.mailerlite.com/article/show/29203-how-to-make-my-own-template).  

Of course, we want to be sure that whatever is done doesn't break some other user.  It might be worth asking MailerLite about this issue.

I just forwarded, as an attachment, the newsletter to a colleague who also has Office 2019 and he sees the same behavior.  While I can't rule out some system/Office setting can correct this issue, it appears that it is a common issue with a default Office setup.  With MS pushing Office 365 it's possible that few of your subscribers use Office 2019.

**Comment by YogiJohn** [comment]
I still see the problem with today's newsletter.  I sent a bug report to MailerLite.  I'll let you know what I hear.

**Comment by kazk** [comment]
Thanks for the update. I got access to MailerLite account so I'll look into if there's anything wrong with the template.

> Around that time I also stopped receiving the Weekly Coding Challenges.

We couldn't send out the newsletter for couple of weeks because the number of subscribers went over the quota and we had trouble upgrading. I think the last one went out on May 28th though.

> I don't see anyway of signing up again for the email list.

I'm not sure what the cause of you not receiving them, but I'll add a way to resubscribe from Account Settings.

**Comment by YogiJohn** [comment]
Thanks.  I received a newsletter today.  And congratulations on the number of subscribers reaching a milestone.

I'm not sure whether I ever looked at it before, but today I also verified that the suspect text is in the newsletter when I access it via webmail, eliminating the small chance that Outlook was modifying the message (e.g. applying a default style).

**Comment by kazk** [comment]
> And congratulations on the number of subscribers reaching a milestone.

Thanks! The number is growing rapidly recently.

Looks like the current newsletter was created using their old "Drag and Drop Editor" long time ago. I don't think I can change the generated code so I need to recreate the email with their new editor and hopefully that doesn't contain the problematic stylesheet rules anymore.

**Comment by YogiJohn** [comment]
The pieces of the puzzle are coming together!  I will update the issue on MailerLite.  Assuming they previously updated their Drag and Drop Editor this may close the issue on their side.

I look forward to seeing what the coming weeks bring (I realize you may be busy and it may take time).

**Comment by kazk** [comment]
Closing because we stopped using MailerLite after having major issues for months. It shouldn't have the same issue because the template was redone, but please let me know if the problem continues.

**Comment by YogiJohn** [comment]
I noticed things were good a week or two ago, but forgot to send you a message.  Thanks for checking.
John
    On Saturday, October 17, 2020, 02:26:50 PM PDT, kazk <notifications@github.com> wrote:  
 
 


Closed #2089.
